libs/core/src/lib/status-indicator/status-indicator.component.ts

```diff
@@ -75,12 +75,6 @@ export class StatusIndicatorComponent implements OnChanges, AfterViewInit, CssCl
     @Input()
     viewBox: string;
 
-    /**
-     * boolean value to define requirement of the label.
-     */
-    @Input()
-    hasLabel: boolean;
-
```

Comment:
Why did you remove it?

Reply:
not required

Reply:
Should be documented in Breaking changes

Reply:
Breaking changes has been updated

Reply:
I don't see it in the PR template

Reply:
<img width="499" alt="Screen Shot 2021-07-22 at 11 01 25 AM" src="https://user-images.githubusercontent.com/39598672/126661766-c51f800a-f9e5-46a1-b17e-e9bfdee93137.png">

I added it